.eslintrc.js

```diff
@@ -2,5 +2,69 @@
 
 module.exports = {
   root: true,
-  extends: '@clark/node'
+  parser: 'babel-eslint',
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  plugins: ['prettier'],
+  extends: ['eslint:recommended', 'prettier'],
+  env: {
+    node: true,
+  },
+  rules: {
+    'prettier/prettier': 'error',
+  },
+
+  overrides: [
+    // node files
+    {
+      files: [
+        '.eslintrc.js',
+        '.prettierrc.js',
+        '.template-lintrc.js',
+        'ember-cli-build.js',
+        'index.js',
+        'testem.js',
+        'blueprints/*/index.js',
+        'config/**/*.js',
+        'tests/dummy/config/**/*.js',
+      ],
+      excludedFiles: [
+        'addon/**',
+        'addon-test-support/**',
+        'app/**',
+        'tests/dummy/app/**',
+      ],
+      parserOptions: {
+        sourceType: 'script',
+      },
+      env: {
+        browser: false,
+        node: true,
+      },
+      plugins: ['node'],
+      extends: ['plugin:node/recommended'],
+    },
+
+    // TypeScript files
+    {
+      files: ['**/*.ts'],
```

Comment:
FWIW, using `overrides` like this means you don't have to use `--ext ts,js` (in ESLint 7).